Mathlib/Order/InitialSeg.lean

```diff
@@ -50,12 +52,15 @@ structure InitialSeg {α β : Type*} (r : α → α → Prop) (s : β → β →
   /-- The order embedding is an initial segment -/
   mem_range_of_rel' : ∀ a b, s b (toRelEmbedding a) → b ∈ Set.range toRelEmbedding
 
--- Porting note: Deleted `scoped[InitialSeg]`
-/-- If `r` is a relation on `α` and `s` in a relation on `β`, then `f : r ≼i s` is an order
-embedding whose range is an initial segment. That is, whenever `b < f a` in `β` then `b` is in the
-range of `f`. -/
+@[inherit_doc]
 infixl:25 " ≼i " => InitialSeg
 
+/-- An `InitialSeg` between the `<` relations of two types. -/
+abbrev InitialSegLT (α β : Type*) [LT α] [LT β] := @InitialSeg α β (· < ·) (· < ·)
+
+@[inherit_doc]
+infixl:25 " ≤i " => InitialSegLT
```

Comment:
Why make it an abbrev?
```suggestion
/-- An `InitialSeg` between the `<` relations of two types. -/
notation:25 alpha:24 " ≤i " beta:25 => @InitialSegLT α β (· < ·) (· < ·)
```
or something like that

Reply:
Nice! I'm not really familiar with the `notation` syntax, but this is much cleaner.